plugins/modules/aci_l3out_extepg.py

```diff
@@ -305,22 +365,51 @@ def main():
             module_object=extepg,
             target_filter={"name": extepg},
         ),
+        child_classes=["fvRsSecInherited", "l3extRsInstPToProfile"],
     )
 
     aci.get_existing()
 
     if state == "present":
-        aci.payload(
-            aci_class="l3extInstP",
-            class_config=dict(
-                name=extepg,
-                descr=description,
-                prefGrMemb=preferred_group,
-                targetDscp=dscp,
-                nameAlias=name_alias,
-            ),
+        child_configs = []
+        if route_control_profiles:
+            profiles = [{key: value} for key, value in route_control_profiles.items()]
```

Comment:
have you considered using list function?

```suggestion
            profiles = list(route_control_profiles.items())
```

Reply:
I'm not sure whether a list of tuples vs a list of dicts really matters here. I made the change regardless.